static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx

```diff
@@ -320,13 +320,16 @@ function AppStoreConnect({
       if (shouldGoNext) {
         setIsLoading(false);
         goNext();
+        return;
       }
+
+      addSuccessMessage(t('An iTunes verification code has been sent'));
     } catch {
       if (shouldGoNext) {
         setIsLoading(false);
       }
       addErrorMessage(
-        t('The iTunes authentication failed. Please check the entered credentials.')
+        t('The iTunes authentication failed. Please check the entered credentials')
```

Comment:
This phrasing seems weird:

> Please check the entered credentials

Maybe

> Please check the given credentials

Reply:
Or 'Please check the provided credentials'

Reply:
I will update the code..thanks for suggestions!